### ISSUE TITLE
setDecorations-renderOptions.after.contentText can not work with chinese

### ISSUE THREAD

**Issue by Qymh** [microsoft/vscode#136813]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.62.1
- OS Version: mac os big sur 11.4

Steps to Reproduce:
raw code
```js
window.onDidChangeActiveTextEditor((e) => {
  if (e) {
    const keysHighlight = window.createTextEditorDecorationType({
      textDecoration: 'none;display:none;',
    });
    e.setDecorations(keysHighlight, [
      {
        range: new Range(0, 12, 0, 23),
        renderOptions: {
          after: {
            contentText: 'mytest',
            color: 'rgba(153, 153, 153)',
          },
        },
      },
    ]);
  }
});
```
i try to replace some text to contentText for an example

the code is `const a = t('Tr-43928a', 'default');`
the image for the code is
![image](https://user-images.githubusercontent.com/29172211/141041313-1c084965-1d6a-4ceb-ae52-f9faaaaacf3b.png)

the expect result is 
![image](https://user-images.githubusercontent.com/29172211/141041453-4e3e4da5-81d4-44b0-a865-fd491daef2cd.png)

in english it is right but if i replace 'default' to '默认' which means 'default' in chinese, it can't work
![image](https://user-images.githubusercontent.com/29172211/141041553-01445f33-0601-4943-9a54-919d1ead805a.png)

I think it's a recent problem
we used it for half a year without any problems

**Comment by gjsjohnmurray** [comment]
Maybe a duplicate of #136622

**Comment by alexdima** [comment]
Let's track in #136622
